cmd/ctl/main.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/etcd-io/auger/pkg/ctl"
+)
+
+func main() {
+	if err := ctl.RootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
```

Comment:
Run an error when added to `cmd` as a subcommand,
So put it here temporarily to verify the feature.

```log
panic: proto: duplicate enum registered: mvccpb.Event_EventType

goroutine 1 [running]:
github.com/golang/protobuf/proto.RegisterEnum({0x105609d6b, 0x16}, 0x140001cd410?, 0x140001cd410)
        /Users/zsm/go/pkg/mod/github.com/golang/protobuf@v1.5.4/proto/registry.go:98 +0xb4
go.etcd.io/etcd/api/v3/mvccpb.init.0()
        /Users/zsm/go/pkg/mod/go.etcd.io/etcd/api/v3@v3.5.13/mvccpb/kv.pb.go:159 +0x38
exit status 2
```